### DISCUSSION TITLE
Disable CSRF

### DISCUSSION THREAD

**Discussion by justayar** [ory/hydra#2763] [Q&A] [answered]
Hi,

Is there any way to disable CSRF all over the Hydra?

Best regards!

**Comment by Benehiko** [comment, marked as answer]
Hi @justayar 

Thanks for reaching out :)

May I ask why you would like to disable CSRF? Hydra isn't just an API, it interacts with your users' browser and uses CSRF to secure those requests. 

Please check out this page from our documentation:
- https://www.ory.sh/hydra/docs/debug/csrf/

**Reply by vinckr** [reply]
Hey @justayar , just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
